drivers/soundwire/stream.c

```diff
@@ -139,10 +139,10 @@ static int sdw_program_slave_port_params(struct sdw_bus *bus,
 					  s_rt->direction,
 					  t_params->port_num);
 	if (!dpn_prop) {
```

Comment:
@RanderWang Could you add comment message here to describe why dpn_prop = null is not an error?